lib/runtime-core/src/typed_func.rs

```diff
@@ -246,6 +297,97 @@ where
     }
 }
 
+impl<'a> ErasedFunc<'a> {
+    /// Creates a polymorphic function.
+    #[allow(unused_variables)]
+    #[cfg(all(unix, target_arch = "x86_64"))]
+    pub fn new_polymorphic<F>(signature: Arc<FuncSig>, func: F) -> Self
+    where
+        F: Fn(&mut vm::Ctx, &[crate::types::Value]) -> Vec<crate::types::Value> + 'static,
+    {
+        use crate::trampoline_x64::*;
```

Comment:
Rather than star imports, can we be explicit on imports (I was having some trouble trying to see where `CallContext` was coming from)